app-accessibility/brltty/brltty-6.1.ebuild

```diff
@@ -56,15 +59,19 @@ DEPEND="
 		x11-libs/libXfixes
 		x11-libs/libXt
 		x11-libs/libXtst
-	)"
-RDEPEND="${DEPEND}
+	)
```

Comment:
I probably wouldn't mess with stuff like this for maintained packages unless it's _wrong_. It's style and we don't want to upset people.